apps/contrib/views.py

```diff
@@ -13,3 +21,30 @@ def get_queryset(self):
         query = self.request.GET.get('q')
         res = sb.search(query, Page, fields=get_search_fields())
         return res
+
+
+def newsletter_view(request):
+    if request.method == 'POST':
+        form = NewsletterForm(request.POST)
+        newsletter_settings = NewsletterSettings.for_request(request)
+        if form.is_valid():
+            configuration = sib_api_v3_sdk.Configuration()
+            configuration.api_key['api-key'] =\
+                newsletter_settings.sendinblue_api_key
+            api_instance = sib_api_v3_sdk.ContactsApi(
+                sib_api_v3_sdk.ApiClient(configuration))
+            create_doi_contact = sib_api_v3_sdk.CreateDoiContact(
+                email=form.cleaned_data['email'],
+                include_list_ids=[newsletter_settings.sendinblue_list_id],
+                template_id=newsletter_settings.sendinblue_template_id,
+                redirection_url=newsletter_settings.redirect_url)
```

Comment:
was wondering if this redirection is being used? or is it needed for CreateDoiContact?